### PR TITLE
Interim updates for the Win2019 dynamic worker rollout

### PR DESCRIPTION
**Pull request by matt-richardson**
Updated docs for the Windows 2019 dynamic worker rollout.
These docs are expected to be modified again within the week - we just want to make sure customers can find information when / if they go looking.

### CODE REVIEW

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 1]
```diff
@@ -23,23 +23,63 @@ Octopus Cloud defines the use of the worker as **finished** when it has been idl
 Each worker is provisioned so that they provide your task a sandbox that is completely isolated from our other customers.
 
 ::: info
-Customers cannot make changes to the default worker pool, or the configuration of the workers themselves. The workers are provisioned using our own service, meaning you don't need to configure anything for them to work in your environment.
+Customers cannot add new workers nor make changes to the configuration of the workers. The workers are provisioned using our own service, meaning you don't need to configure anything for them to work in your environment.
 :::
 
+### Types of Dynamic Worker
+
+Each dynamic worker pool can specify the worker image used. As of July 2020, `WindowsDefault` is the default, which is Windows Server Core 2016. `Windows2019` (Windows Server Core 2019) worker images are rolling out. Once the rollout is complete, Windows Server Core 2016 will be marked as deprecated.
```
Comment (wordlee) [inline docs/infrastructure/workers/dynamic-worker-pools.md:31]: Picky edit:
```suggestion
Each dynamic worker pool can specify the worker image used. As of July 2020, `WindowsDefault` is the default, which is Windows Server Core 2016. `Windows2019` (Windows Server Core 2019) worker images are rolling out. After the rollout is complete, Windows Server Core 2016 will be marked as deprecated.
```

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 2]
```diff
@@ -23,23 +23,63 @@ Octopus Cloud defines the use of the worker as **finished** when it has been idl
...
+
+Editing a dynamic worker pool allows you to modify the image used.
+
+When an image is marked as deprecated, you will start to see warnings in the Octopus UI, and in the deployment log. After a suitable deprecation period, deployments will start to fail if they target an image that has hit end-of-life.
```
Comment (wordlee) [inline docs/infrastructure/workers/dynamic-worker-pools.md:35]:
```suggestion
When an image is marked as deprecated, you will start to see warnings in the Octopus UI, and in the deployment log. After a suitable deprecation period, deployments will start to fail if they target an image that has hit end-of-life.
```
```suggestion
When an image is marked as deprecated, you will see warnings in the Octopus UI, and in the deployment log. After a suitable deprecation period, deployments will start to fail if they target an image that has hit end-of-life.
```

Comment (wordlee) [inline docs/infrastructure/workers/dynamic-worker-pools.md:35]:
```suggestion
When an image is marked as deprecated, you will see warnings in the Octopus UI, and in the deployment log. After a suitable deprecation period, deployments will start to fail if they target an image that has hit end-of-life.
```

Comment (wordlee) [reply docs/infrastructure/workers/dynamic-worker-pools.md:35]: Also, do we have something more concrete than. "suitable deprecation period"? A month, three months?

Comment (matt-richardson) [reply docs/infrastructure/workers/dynamic-worker-pools.md]: > Also, do we have something more concrete than. "suitable deprecation period"? A month, three months?

It did say "6 months", but we were hesitant of painting ourselves into a corner. I might check that one with @MJRichardson

Comment (matt-richardson) [reply docs/infrastructure/workers/dynamic-worker-pools.md]: I'm going to merge away for now, as we're going to be updating this within the week anyway.

**Pull request merged**